Client/Frontend/Home/LogoHeader/HomeLogoHeaderCell.swift

```diff
@@ -9,32 +9,42 @@ import UIKit
 
 class HomeLogoHeaderCell: UICollectionViewCell, ReusableCell {
     private struct UX {
+        static let iPadAdjustment: CGFloat = -40
         struct Logo {
-            static let imageSize: CGFloat = 40
-            static let topConstant: CGFloat = 32
+            static let iPhoneImageSize: CGFloat = 40
+            static let iPadImageSize: CGFloat = 75
+            static let iPhoneTopConstant: CGFloat = 32
+            static let iPadTopConstant: CGFloat = 70
             static let bottomConstant: CGFloat = -10
         }
 
         struct TextImage {
-            static let imageWidth: CGFloat = 70
-            static let imageHeight: CGFloat = 40
-            static let leadingConstant: CGFloat = 9
+            static let iPhoneWidth: CGFloat = 70
+            static let iPadWidth: CGFloat = 133
+            static let iPhoneLeadingConstant: CGFloat = 9
+            static let iPadLeadingConstant: CGFloat = 17
             static let trailingConstant: CGFloat = -15
         }
     }
 
     typealias a11y = AccessibilityIdentifiers.FirefoxHomepage.OtherButtons
 
     // MARK: - UI Elements
-    lazy var logoImage: UIImageView = .build { imageView in
+    private lazy var logoImage: UIImageView = .build { imageView in
         imageView.image = UIImage(imageLiteralResourceName: ImageIdentifiers.homeHeaderLogoBall)
         imageView.contentMode = .scaleAspectFit
         imageView.accessibilityIdentifier = a11y.logoImage
     }
 
-    lazy var logoTextImage: UIImageView = .build { imageView in
+    private lazy var logoTextImage: UIImageView = .build { imageView in
         imageView.contentMode = .scaleAspectFit
         imageView.accessibilityIdentifier = a11y.logoText
+        imageView.accessibilityLabel = AppName.shortName.rawValue
+        imageView.isAccessibilityElement = true
```

Comment:
curious - since the text and the logo are both images, would it be preferable to group the two elements as one accessibility element? Grouping makes sense to me as we're more explicit about the accessibility value for both images, but not sure how much impact it really has. Just a thought that cross my mind!

Reply:
Oooohhh, I meant to add this to the container view that groups them, but I forgot. Good catch!